packages/api/src/server/analytics/service/analytics.service.ts

```diff
@@ -13,10 +15,14 @@ export class AnalyticsService {
 
   private static readonly channel: string = 'activity_stream';
 
+  private static readonly ephemeral: string = 'ephemeral';
+
   private static readonly defaultSkip: number = 0;
 
   private static readonly defaultLimit: number = 100;
 
+  private static readonly defaultDays: number = 120;
```

Comment:
@ayushsatyam146 @soumyadip007 Should we change all of these variables to capital?